### ISSUE TITLE
Feature Request: Ability to identify meeting host server via API

### ISSUE THREAD

**Issue by mailalaki** [blindsidenetworks/scalelite#725]
Hi, when we use scalellite with a number of servers under its management, there is a gap so that we can easily identify the server where that meeting is hosted via "getMeetingInfo" or "getMeeting".
This can be easily done by rewriting the "create" call that goes from the scalelite to the server selected by the loadbalancer. So that the "Server URL" is added in the meta section of that session.

for example:
![scalelite](https://user-images.githubusercontent.com/13551603/140056542-349fada6-351a-43b0-9235-fd0a1415994d.PNG)

Given the usefulness of this information for use in applications, please add such a feature.

**Comment by amalrails** [comment]
@mailalaki why exactly do you need this feature? if its just for finding a meeting's server  you can use `./bin/rake meetings:list["meetingId1:meetingId2:meetingId3"]`

**Comment by mailalaki** [comment]
> @mailalaki why exactly do you need this feature? if its just for finding a meeting's server you can use `./bin/rake meetings:list["meetingId1:meetingId2:meetingId3"]` https://github.com/blindsidenetworks/scalelite#list-allspecific-meetings-running-in-bigbluebutton-servers

Yes, but when we need to inquire about the status of a meeting in the third-party application in the simplest possible way, it is difficult to work with the method you suggest, and the best option is to use the api.
The method I requested was a suggested method. It would be very useful if in any way the information of the bigbluebutton server that hosted a meeting could be obtained from the scalelite server by requesting the api.
However, our proposed method has another advantage. The advantage is that the information on which server this meeting was hosted is stored in the metadata along with it, and this can be useful for future uses. For example, when we want to filter recordings or track raw data of recordings.

**Comment by mailalaki** [comment]
Merely to remind!
